tests/simpleharness/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is the manifest that is usually used in .NET Framework to elevate your application to run as Administrator, but it does not work on .NET Core (at least for now). I figured I would give it a try. I can remove it if you think it is not necessary.

---
In reply to: [100349646](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100349646) [](ancestors = 100349646)